tests/test_main.py

```diff
@@ -133,11 +133,24 @@ def test_ruleset():
                     """
                 )
             ),
+            capa.rules.Rule.from_yaml(
+                textwrap.dedent(
+                    """
+                    rule:
+                        meta:
+                            name: process rule
+                            scope: process
```

Comment:
process should not be valid as a static-flavor scope. i see that you haven't split out the static/dynamic flavors yet, so i realize you can't test this quite yet.